frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -104,7 +103,6 @@ class Dashboard extends Component {
   };
 
   static defaultProps = {
-    isEditable: true,
```

Comment:
I can't find where we pass this prop, so it's safe to say that it's unused. I hope the CI will help confirm that.

Reply:
I'd not remove anything you're not confident about 😄 first migrate to TS, then delete props

Reply:
I was like 90% confident. Text search and stuff would go a long way. I've checked I think 3 other components that use Dashboard's `...this.props` which also includes `isEditable` and all those components don't utilize `isEditable` at all.

Reply:
It's introduced in https://github.com/metabase/metabase/pull/4123, which I also couldn't find if we used to pass any prop either. Feels like it's been set to be a defaultProps from a start.